phpBB/styles/prosilver/template/viewtopic_body.html

```diff
@@ -80,7 +80,7 @@ <h2 class="poll-title"><!-- EVENT viewtopic_body_poll_question_prepend -->{POLL_
 			<fieldset class="polls">
 			<!-- BEGIN poll_option -->
 				<!-- EVENT viewtopic_body_poll_option_before -->
-				<dl class="<!-- IF poll_option.POLL_OPTION_VOTED -->voted<!-- ENDIF --><!-- IF poll_option.POLL_OPTION_MOST_VOTES --> most-votes<!-- ENDIF -->"<!-- IF poll_option.POLL_OPTION_VOTED --> title="{L_POLL_VOTED_OPTION}"<!-- ENDIF --> data-poll-option-id="{poll_option.POLL_OPTION_ID}">
+				<dl class="<!-- IF poll_option.POLL_OPTION_VOTED -->voted<!-- ENDIF --><!-- IF poll_option.POLL_OPTION_MOST_VOTES --> most-votes<!-- ENDIF -->"<!-- IF poll_option.POLL_OPTION_VOTED --> title="{L_POLL_VOTED_OPTION}"<!-- ENDIF --> data-alt-text="{LA_POLL_VOTED_OPTION}" data-poll-option-id="{poll_option.POLL_OPTION_ID}">
```

Comment:
I have used LA because the string is used from JS, and because the (only) example of use of the same construct in phpbb also uses LA:
https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/styles/prosilver/template/memberlist_body.html#L28

But if you tell me I must change it to L_, I will do.